crates/context/config/src/client/env/proxy/types/starknet.rs

```diff
@@ -289,19 +331,26 @@ impl From<Vec<ProposalAction>> for StarknetProposalActionWithArgs {
 impl From<StarknetProposalActionWithArgs> for ProposalAction {
     fn from(action: StarknetProposalActionWithArgs) -> Self {
         match action {
-            StarknetProposalActionWithArgs::ExternalFunctionCall(contract, selector, calldata) => {
-                ProposalAction::ExternalFunctionCall {
-                    receiver_id: format!("0x{}", hex::encode(contract.to_bytes_be())),
-                    method_name: format!("0x{}", hex::encode(selector.to_bytes_be())),
-                    args: calldata
-                        .iter()
-                        .map(|felt| format!("0x{}", hex::encode(felt.to_bytes_be())))
-                        .collect::<Vec<_>>()
-                        .join(","),
-                    deposit: 0,
-                    gas: 0,
-                }
-            }
+            StarknetProposalActionWithArgs::ExternalFunctionCall(
+                contract,
+                selector,
+                amount,
+                calldata,
+            ) => ProposalAction::ExternalFunctionCall {
+                receiver_id: format!("0x{}", hex::encode(contract.to_bytes_be())),
+                method_name: format!("0x{}", hex::encode(selector.to_bytes_be())),
```

Comment:
these are the same as `.to_string()`, right? just checking

```suggestion
                receiver_id: contract.to_string(),
                method_name: selector.to_string(),
```